client/src/Editor/App.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
Nice! I thought of using react but was not sure if it is too much for me to do in 2 hours.  Great job!